src/concrete/ml/onnx/convert.py

```diff
@@ -159,6 +159,7 @@ def get_equivalent_numpy_forward_from_torch(
 def get_equivalent_numpy_forward_from_onnx(
     onnx_model: onnx.ModelProto,
     check_model: bool = True,
+    _auto_rounder: Optional[Callable[..., Any]] = None,
```

Comment:
I don't think it's customary to prefix arguments with "_"

Reply:
I added it as a private variable just to facilitate my tests.
I removed it.